doc/ADMIN.md

```diff
@@ -1,4 +1,13 @@
-* Email required for administation (uses "magic links" instead of admin passwords)
+Please navigate to https://__DOMAIN____PATH__ to subscribe as administrator on __APP__!
 
-* You should manually configure the app after installation.
-* You should edit __INSTALL_DIR__/run_fider to add environment variables, e.g for GitHub integration
+You can register using an email from *any* email provider. You will receive an email with a login link inside.
+
+Then, you can configure the app.
```

Comment:
So hmmm sorry for being pedantic, but if this is something that is expected to be told to the admin, and done right after installation, then this should go in `POST_INSTALL.md`. Otherwise people probably won't see this message until they browse to the webadmin and scroll down on the app's info page

The other bit about `custom_fider_env` can stay in `ADMIN.md`, because as far as I understand, because this is a general admin tip about advanced stuff

Reply:
> sorry for being pedantic

No worries, i wasn't entirely sure about what was shown where :)

> The other bit about custom_fider_env can stay in ADMIN.md, because as far as I understand, because this is a general admin tip about advanced stuff

Hmm yeah but it could be a good idea of keeping all this info grouped

Reply:
Eerrr  but the point of the message in `POST_INSTALL.md` is that you can "dismiss" it once you've read and taken the action mentionned, which is meant to be a one-time thing to finalize the install ... @_@

Reply:
Ah. So i should duplicate the info in post install and admin.md ? 

IMHO the post-install info should be visible in the app page : what if the yunohost admin dashboard crashed at the end of the install ? What if the admin want to verify they did the proper thing ?

Reply:
>IMHO the post-install info should be visible in the app page : what if the yunohost admin dashboard crashed at the end of the install ? What if the admin want to verify they did the proper thing ?

(yes, the post-install is already visible in the app page until the admin clicks the "dismiss" button. This was designed this way for example because the postinstall instructions may contain secrets like database password, which you don't want to show forever in the admin interface once the admin did the initial configuration)